CMakeLists.txt

```diff
@@ -22,7 +26,7 @@ add_compile_options(-Wno-unused-parameter -Wno-missing-field-initializers
 add_compile_options(-g -O2 -W -Wall -Wextra)
 add_compile_options(
   $<$<COMPILE_LANGUAGE:CXX>:-Wno-reorder>
-  $<$<COMPILE_LANGUAGE:CXX>:-std=c++11>
+  $<$<COMPILE_LANGUAGE:CXX>:-std=c++14>
```

Comment:
C++14 can't be used because OpenDingux only has C++11 support.

Reply:
Conditionned to the `APPLE` cmake var now, as right now only MacOS doesn't have it on C++11 too